### DISCUSSION TITLE
boost-python, python version

### DISCUSSION THREAD

**Discussion by owillebo** [microsoft/vcpkg#18629] [Q&A]
How can I set the Python version for installing boost-python?

I need boost-python for Python 3.8.

**Comment by Osyotr** [comment]
`boost-python` depends on port `python3`, so if you want to use specific python version, you should [override](https://github.com/microsoft/vcpkg/blob/master/docs/examples/versioning.getting-started.md#overrides) it in your manifest.
I.e.:
```
{
    "name": "my-project",
    "version": "1.0.0",
    "builtin-baseline": "b60f003ccf5fe8613d029f49f835c8929a66eb61",
    "dependencies": [
        "boost-python",
        "python3"
    ],
    "overrides": [
        {
            "name": "python3",
            "version": "3.8.0"
        }
    ]
}
```
